### DISCUSSION TITLE
Do I need to resend my student cart to get Github education pack ? If yes how much time it would take ?

### DISCUSSION THREAD

**Discussion by Christ7785** [community/community#144682] [GitHub Education] [answered]
### Hi everyone,

I have been approved years ago but this year it didnt ask me to resend yet. Do I need to send it ?

**Comment by Ozomozyan** [comment, marked as answer]
If it didnt ask you yet. Its most likely that its not your time to resend your student cart yet and if it asks you someday, apperantly it takes 3 to 5 days for them to approve or give you acces to your benefits
